website/docs/policy/policy-configuration.mdx

```diff
@@ -254,26 +260,25 @@ Deploying `app-a` in another namespace other than `flux-system` won't be affecte
   - <em>Any application or resource in namespace `flux-system` replica_count must be `>= 2`</em>
   - <em>Also it will be affected by `my-config-1` for `owner` configuration parameter `owner: owner-1`</em>
 
-
-**Note** 
-
-- You can use one or more policies as the following example
-
-  ```yaml
-  apiVersion: pac.weave.works/v2beta2
-  kind: PolicyConfig
-  metadata:
-    name: my-app-config
-  spec:
-    match:
-      resources:
-        name: my-deployment
-        kind: Deployment
-    config:
-      weave.policies.policy-1:
-        params:
-          replica_count: 3
-      weave.policies.policy-2:
-        params:
-          run_as_root: true
-  ```
+	:::note
+	You can use one or more policies as the following example
```

Comment:
we could adjust this 

<img width="1164" alt="Screenshot 2023-02-21 at 12 43 49" src="https://user-images.githubusercontent.com/12957664/220348008-4af9e44b-3f99-467e-9101-1ef499dfca0e.png">